### PR TITLE
Adjust layout of commercial components on mobile

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Essentially made sure components fit in a 400px high window by hiding and moving elements around:
picture 2

Now
picture 1

@uplne @piuccio

### CODE REVIEW

**Comment by piuccio** [comment]
isn't it too narrow to have two cards on one line?
Anyway it looks much better than the previous solution.

Would it be too hard to bottom align `book now` buttons?

**Comment by uplne** [comment]
Debugging is twice as hard as writing the code in the first place. Therefore, if you write the code as cleverly as possible, you are, by definition, not smart enough to debug it.

**Comment by regiskuckaertz** [comment]
I always keep an ace in my sleeve :-)

@piuccio I could using `margin-top: auto` but then I would lose the minimum top margin between it and what's before.

**Comment by piuccio** [comment]
maybe flexbox?

**Comment by regiskuckaertz** [comment]
im already using it 😃

File: static/src/stylesheets/module/commercial/_adverts-soulmates.scss [hunk 1]
```diff
@@ -96,6 +108,7 @@
 
 .advert-blended--soulmates {
     background: $soulmates-background;
+    box-sizing: border-box;
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/_adverts-soulmates.scss]: Do you actually understand what `border-box` does?

Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts-soulmates.scss]: In this case the combination of `padding` and `flex-basis` drives Chrome crazy and prevents it from breaking words; `border-box` solves this.

Comment (uplne) [inline static/src/stylesheets/module/commercial/_adverts-soulmates.scss]: Great

**Comment by uplne** [comment]
👍

**Pull request merged**